internal/app/tfsec/adapter/aws/cloudwatch/adapt.go

```diff
@@ -6,5 +6,35 @@ import (
 )
 
 func Adapt(modules []block.Module) cloudwatch.CloudWatch {
-	return cloudwatch.CloudWatch{}
+	return cloudwatch.CloudWatch{
+		LogGroups: adaptLogGroups(modules),
+	}
+}
+
+func adaptLogGroups(modules []block.Module) []cloudwatch.LogGroup {
+	var logGroups []cloudwatch.LogGroup
+	for _, module := range modules {
+		for _, resource := range module.GetResourcesByType("aws_cloudwatch_log_group") {
+			logGroups = append(logGroups, adaptLogGroup(resource))
+		}
+	}
+	return logGroups
+}
+
+func adaptLogGroup(resource block.Block) cloudwatch.LogGroup {
+	nameAttr := resource.GetAttribute("name")
+	nameVal := nameAttr.AsStringValueOrDefault("", resource)
+
+	KMSKeyIDAttr := resource.GetAttribute("kms_key_id")
+	KMSKeyIDVal := KMSKeyIDAttr.AsStringValueOrDefault("", resource)
```

Comment:
I think we could just do `KMSKeyIDVal := resource.GetAttribute("kms_key_id").AsStringValueOrDefault("", resource)` to make it even more test